### PR TITLE
Create 1-1-tr.srt

### PR DESCRIPTION
**Pull request by omergulen**
Turkish subtitle for try-python 1-1.
@snychka done !

### CODE REVIEW

**Comment by omergulen** [comment]
@snychka hey mate!

**Comment by snychka** [comment]
Cool!  Thanks for the @ .  Just letting you know I'm aware .

fyi (and til):  I didn't get you first one as I think that's in your repo.  I did get notified by the 2nd one, so in this case you need to @ in a comment on your PR.  Makes sense

**Comment by snychka** [comment]
Just started checking into this now.  `-tr` isn't supported currently by our code.  Uncertain if this'll be an easy addition or not.  Will need to ask a dev about this.  Will get back to you!

**Comment by omergulen** [comment]
Okay, but it should be very easy to do it. If you would merge it, I would countinue to translate.

**Comment by snychka** [comment]
Hey @omergulen !

tl;dr:  there will likely be a wait, and I'll ping you and merge this when it's ready to go!

---

I thought the change was surprisingly hard, despite the fact it looked easy.

The change is in fact easy.  Good news!

It affects our video player though which is key, so I need to make an internal PR of my own and when the lead dev has the time, he can approve my PR and publish it, but that depends on many things.  I don't know the details, and they're private anyway, but I'm sure there're a lot of other considerations.

So this could take a while, and I currently have no eta for this.  I will ask later if this takes longer (i.e., still not merged next week).

I'll make the PR later today.  Your current change looks good, btw.

Don't work on other levels for now.  I'm not going to merge your PR until  the support for `-tr` is added to the video player.

I'll ping you and likely merger your PR when the support is published!

---

<details>
<summary>ignore.  for my reference later, for a diff</summary>


```
one
00: 00: 00,000 -> 00: 00: 01,206
[MUSIC PLAYING]

2nd
00: 00: 01,206 -> 00: 00: 03,590
[SINGING] Now a little bit,
Python learning time

3
00: 00: 03,590 -> 00: 00: 06,830
String print, daily
Mathematics, 
4
00: 00: 06,830 -> 00: 00: 09,340
On the horizon.

5
00: 00: 09,340 -> 00: 00: 12,570
Try Ptyhon's simple managers

6
00: 00: 12,570 -> 00: 00: 17,040
True / False and conditions
Start using it.

7
00: 00: 17,040 -> 00: 00: 18,860
Welcome to Try Python.

8
00: 00: 18,860 -> 00: 00: 22,550
I am Sarah Buchanan, this course
We will learn the basics of Python

9
00: 00: 22,550 -> 00: 00: 25,690
Let's start with level 1:

10
00: 00: 25,690 -> 00: 00: 30,630
Birds and coconuts,
Representing mathematics and variables.

11th
00: 00: 30,630 -> 00: 00: 33,240
In our Python journey
Before starting,

12
00: 00: 33,240 -> 00: 00: 36,540
Why Pyhton is so
I'll pass them over perfectly.

13
00: 00: 36,540 -> 00: 00: 41,060
Python is easy to learn and
Easy to read,

14
00: 00: 41,060 -> 00: 00: 43,480
But in fact
A strong lie,

15
00: 00: 43,480 -> 00: 00: 47,420
Beginners with this feature
And it is very nice for experts.

16
00: 00: 47,420 -> 00: 00: 50,440
In addition,
Has a library support,

17
00: 00: 50,440 -> 00: 00: 52,110
That is,
When doing a data analysis,


18
00: 00: 52,110 -> 00: 00: 55,620
While creating a game or
For a lot of things like

19
00: 00: 55,620 -> 00: 00: 59,760
You can download the library from the internet.

20
00: 00: 59,760 -> 00: 01: 02,480
It is important to learn Python
Along with all this reason,

21
00: 01: 02,480 -> 00: 01: 05,170
let's start.

22
00: 01: 05,170 -> 00: 01: 07,620
Python is an interpretable idiom,

23
00: 01: 07,620 -> 00: 01: 10,360
And an interpretive shell
Here it appears,

24
00: 01: 10,360 -> 00: 01: 13,470
Just like a calculator

25
00: 01: 13,470 -> 00: 01: 15,210
Numbers or
We can write commands.

26
00: 01: 15,210 -> 00: 01: 19,240
For example,
35 writers,

27
00: 01: 19,240 -> 00: 01: 21,990
He will immediately return it to us.

28
00: 01: 21,990 -> 00: 01: 24,960
You see the >>> sign
Always understand

29
00: 01: 24,960 -> 00: 01: 27,970
We are entering code in the interpreter,

30
00: 01: 27,970 -> 00: 01: 32,250
And the green arrow is the code we wrote
It returns the result.

31
00: 01: 32,250 -> 00: 01: 36,080
For example, if 30.5 writes,

32
00: 01: 36,080 -> 00: 01: 37,335
We get the same back.

33
00: 01: 40,160 -> 00: 01: 43,480
Any interpreter entered
A mathematical expression

34
00: 01: 43,480 -> 00: 01: 48,540
Simple operators like addition (+)
.

35
00: 01: 48,540 -> 00: 01: 52,570
For example, if you write 3 + 5,

36
00: 01: 52,570 -> 00: 01: 55,300
We have 8 results.

37
00: 01: 55,300 -> 00: 02: 00,820
Also some other simple
Operators; extraction,

38
00: 02: 00,820 -> 00: 02: 06,970
impact,
Division, exponential,

39
00: 02: 06,970 -> 00: 02: 11,100
**,
And negative numbers,

40
00: 02: 11,100 -> 00: 02: 14,950
You only need to put minus (-).

41
00: 02: 14,950 -> 00: 02: 18,410
Python has two numbers;
The whole number called "int"

42
00: 02: 18,410 -> 00: 02: 22,230
The other called "float"
Decimal numbers.

43
00: 02: 22,230 -> 00: 02: 25,590
If you do not see a decimal
All numbers are integers (int),

44
00: 02: 25,590 -> 00: 02: 29,714
Terms such as int and float

45
00: 02: 29,714 -> 00: 02: 30,630
We will use it throughout the course.

46
00: 02: 33,170 -> 00: 02: 36,530
All other programming languages
Also in Python

47
00: 02: 36,530 -> 00: 02: 39,130
The order of operators is important.

48
00: 02: 39,130 ​​-> 00: 02: 41,910
Earned (5 + 7) is calculated before

49
00: 02: 41,910 -> 00: 02: 45,840
Because it is in parentheses.

50
00: 02: 45,840 -> 00: 02: 49,200
Thereafter, 12 * 3 operations are performed

51
00: 02: 49,200 -> 00: 02: 52,270
And we reach 36 results.

52
00: 02: 52,270 -> 00: 02: 55,420
If you see this icon,

53
00: 02: 55,420 -> 00: 02: 57,580
Backstage steps
We will show.

54
00: 02: 57,580 -> 00: 03: 01,430
On the right side, except for the brackets,

55
00: 03: 01,430 -> 00: 03: 03,010
7 * 3 is calculated first.

56
00: 03: 03,010 -> 00: 03: 09,380
Steps backstage: from 5 + 21
Result 26.

57
00: 03: 09,380 -> 00: 03: 13,470
To memorize the sequence
We can use PÜÇMEÇ,

58
00: 03: 13,470 -> 00: 03: 16,790
Bracket, Base, Multiplication,

59
00: 03: 16,790 -> 00: 03: 20,550
Partitioning, Addition (Addition), Subtraction.

60
00: 03: 20,550 -> 00: 03: 24,470
Now our Python Mathematics knowledge
Daily problems Monty Python

61
00: 03: 24,470 -> 00: 03: 28,400
Practice in the world:

62
00: 03: 28,400 -> 00: 03: 29,600
Can a Swallow Carry An Indian Walnut?

63
00: 03: 29,600 -> 00: 03: 34,170
The first thing we know,
A swallow is 60 grams.

64
00: 03: 34,170 -> 00: 03: 36,480
Wind speed and other factors
together

65
00: 03: 36,480 -> 00: 03: 38,780
1/3 of the weight of

66
00: 03: 38,780 -> 00: 03: 42,730
.

67
00: 03: 42,730 -> 00: 03: 47,470
By dividing 60 by 3,

68
00: 03: 47,470 -> 00: 03: 50,320
The prairie can carry 20 grams
See,

69
00: 03: 50,320 -> 00: 03: 54,400
But coconut
It's 1450 grams.

70
00: 03: 54,400 -> 00: 03: 58,620
Apparently, the swallow
Can not carry a coconut.

71
00: 03: 58,620 -> 00: 04: 01,610
Then, how many swallow
Let's see if he can carry it.

72
00: 04: 01,610 -> 00: 04: 05,750
The weight of coconut,
1450 grams,

73
00: 04: 05,750 -> 00: 04: 10,040
Dividing the weight of the fragile transport,
60/3.

74
00: 04: 10,040 -> 00: 04: 12,560
This is us 
How many swallow is needed

75
00: 04: 12,560 -> 00: 04: 14,800
He will say.

76
00: 04: 14,800 -> 00: 04: 19,089
We finally got 8.06,
A very low number.

77
00: 04: 19,089 -> 00: 04: 21,630
It seems that somewhere,
We just stuffed something.

78
00: 04: 21,630 -> 00: 04: 26,300
What we calculated first
If we have a look,

79
00: 04: 26,300 -> 00: 04: 28,520
Made a mistake
We can see that we can be.

80
00: 04: 28,520 -> 00: 04: 33,250
First of all,
We wanted to do,

81
00: 04: 33,250 -> 00: 04: 35,770
To fix it
It will suffice to add a few parentheses.

82
00: 04: 38,730 -> 00: 04: 40,680
When we add parentheses 
To transport coconut

83
00: 04: 40,680 -> 00: 04: 44,110
Required 72.5 fragrance
We will find

84
00: 04: 44,110 -> 00: 04: 46,900
, So much swallow.

85
00: 04: 50,490 -> 00: 04: 52,510
Now it is your turn;
Duties

86
00: 04: 52,510 -> 00: 04: 55,540
Write your first Python program.

87
00: 04: 55,540 -> 00: 04: 57,360
Have fun.
```


</details>

**Comment by omergulen** [comment]
Okay! I'll be waiting. @snychka

**Comment by snychka** [comment]
@omergulen , howdy, and it's been merged!

**Comment by omergulen** [comment]
Thank you, then I'll start to translate others!
